### DISCUSSION TITLE
NIST RSE Heat Flux Validation Data

### DISCUSSION THREAD

**Discussion by johodges** [firemodels/fds#12153] [General]
I have been looking into the interaction of pyrolysis and flame extinction in under-ventilated compartment fires as part of our railcar work. I came across an IAFSS paper Kevin and Jason wrote in 2008 using the NIST RSE data. We include the species concentration and temperature measurements from these cases in the validation guide, but omit the floor heat flux data. Is there a reason these data are omitted in the current V&V guide?

https://publications.iafss.org/publications/fss/9/117/view/fss_9-117.pdf

**Comment by drjfloyd** [comment]
I don't recall any specific reason. Probably we were just more focused on the combustion aspect (temp and species) when putting these in the Validation Guide.  We also don't have the velocity data plotted.

**Comment by mcgratta** [comment]
Laziness, probably. Did we include the floor points in the input files? If so, we could make some plots. Maybe it was just a plot overload and we decided to choose the walls and ceiling.

**Comment by drjfloyd** [comment]
Looked at RSE_01.fds and we have the two HF gauges on the floor and all the door velocities.

**Comment by johodges** [comment]
Thanks Jason. I can add them to datplot and see how the measurements look. I am trying to get a sense for how much Xr is changing in these underventilated cases. I have been playing around with ramping Xr using a global o2 concentration in our railcar to reduce the heat flux since we are currently over-predicting the HRR .

**Comment by drjfloyd** [comment]
Are you doing that with RAMP_CHI_R and using CTRL_ID or DEVC_ID to reference the global O2 concentration?

**Reply by johodges** [reply]
Yep, here are the lines I am using.

&DEVC ID='OxygenConcentration', XB=0.72,2.61,0.42,1.2,0.3048,0.6, QUANTITY='VOLUME FRACTION', SPEC_ID='OXYGEN', SPATIAL_STATISTIC='VOLUME MEAN', /

&REAC ID='REAC_FUEL',
      FUEL='CELLULOSE',
      C=1, 
      H=0.65, 
      O=0.5, 
      SOOT_YIELD=0.018, 
      CO_YIELD=0.093, 
      RADIATIVE_FRACTION=1.,
      AUTO_IGNITION_TEMPERATURE=600.0,
      RAMP_CHI_R='Cellulose_Xr_Ramp',
      
/

&RAMP ID='Cellulose_Xr_Ramp', T=0.0, F=0.10, DEVC_ID='OxygenConcentration', /
&RAMP ID='Cellulose_Xr_Ramp', T=0.21, F=0.30, /

**Reply by drjfloyd** [reply]
Nice, good to see that capability being used!

**Comment by johodges** [comment]
One thing I am seeing with these cases is the results are highly sensitive to how the flame was tilting from the pan. Example here from test 4. You can see in the experiment the flame was directly exposing the rear gauge which is not occurring in FDS.

![image](https://github.com/firemodels/fds/assets/17127980/799dbe12-8613-4457-9f48-22f8102f2c18)

Adding these tests is going to have a pretty big impact on the uncertainty statistics for heat flux. In FDS 6.8 we had a bias of 0.97 and a sigma_m of 0.33. For these tests alone I am seeing a bias of 1.07 with a sigma_M of 0.52.

![image](https://github.com/firemodels/fds/assets/17127980/db53445e-9fe8-4a05-bbed-bf3194cef956)

**Reply by drjfloyd** [reply]
Assuming a gauge temperature of ~30 C, gauge emissivity of 0.95, and h=10 W/m/K, getting ~240 kW/m2 (peak exp) and ~150 kW/m2 (peak FDS) takes effective gas temperatures of ~1160 C and ~1000 C a 12 % difference.  These are the temperatures seen in the temperature plot. May not be as much the flame position as other errors in the temperature.

**Comment by johodges** [comment]
Added these data to the V&V guide in https://github.com/firemodels/fds/pull/12154. Here are the updated parts of the guide:

[FDS_Validation_Guide_rse_hf_only.pdf](https://github.com/firemodels/fds/files/12738787/FDS_Validation_Guide_rse_hf_only.pdf)
